tests/dbus/test_dbus_client.cpp

```diff
@@ -239,6 +239,16 @@ void CheckNat64(ThreadApiDBus *aApi)
 #endif
 }
 
+void CheckTelemetryData(ThreadApiDBus *aApi)
+{
+    OTBR_UNUSED_VARIABLE(aApi);
+#if OTBR_ENABLE_TELEMETRY_DATA_API
+    std::vector<uint8_t> responseTelemetryDataBytes;
+
```

Comment:
I'm a bit concerned that the test case doesn't cover the situation that there's any payload in the message.

Reply:
I added the wpan_stats proto payload and assert its content now in the test case.